lib/syskit/cli/log_runtime_archive_main.rb

```diff
@@ -114,6 +114,51 @@ def transfer_server( # rubocop:disable Metrics/ParameterLists
                               implicit_ftps)
             end
 
+            desc "watch_ensure_free_space", "watches the ensure free space process"
+            option :period,
+                   type: :numeric, default: 10, desc: "polling period in seconds"
+            option :max_size,
+                   type: :numeric, default: 10_000, desc: "max log size in MB"
```

Comment:
Right. So this is here because the archiver class needs it at construction time. But it's relevant only for archiving.

Now that we have other uses for it, it should be passed as argument to the archive methods instead of given to .new

Reply:
Going to refactor this in a PR on top of this one.

Reply:
https://github.com/rock-core/tools-syskit/pull/458